packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -213,6 +213,7 @@ const ClinicalResultBloodPressure = (props) => {
           >
             <Observation
               key={`Observation-Diastolic-${diastolic.eventId}`}
+              className={!compareStatusIsInError.diastolic && diastolic.interpretation ? cx('diastolic-intrepretation') : null}
```

Comment:
```suggestion
              className={cx({ 'diastolic-intrepretation': !compareStatusIsInError.diastolic && diastolic.interpretation })}
```

Reply:
classnames/bind will only apply the class name when it evaluates to true: https://www.npmjs.com/package/classnames#alternate-bind-version-for-css-modules-

Reply:
Had it this was but was afraid the prop would try and spread the empty string. Documentation to key not being added when value is falsey is helpful.

Reply:
Fixed here https://github.com/cerner/terra-clinical/pull/637/commits/788a6eed719217b512d85dbc8c1fa1c37d747e50#diff-13926e711e02256769a4612c0d911268R216